src/components/views/settings/ChangePassword.tsx

```diff
@@ -399,7 +399,7 @@ export default class ChangePassword extends React.Component<IProps, IState> {
                             <PassphraseField
                                 fieldRef={(field) => (this[FIELD_NEW_PASSWORD] = field)}
                                 type="password"
-                                label={_td("New Password")}
+                                label={_td("New password")}
```

Comment:
I think we should avoid making this change unless we change other references to `New Password` in the application (which should be in a separate PR)

Reply:
I would agree. I am going to create an issue to track it.

Reply:
The issue was created here: https://github.com/vector-im/element-web/issues/25148. It looks like that capitalization on that area has not been very strict.

Reply:
It was reverted.